### PR TITLE
#13 updated dependencies

### LINKED ISSUE #13: Dependency update
Dependency update

### PR DESCRIPTION
**Pull request by ckunki**
fixes #13

### CODE REVIEW

File: doc/changes/changes_2.0.2.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Test utilities for `java.util.logging` 2.0.2, released 2022-08-30
+
+Code name: Dependency updates
+
+## Summary
+
+Updated dependencies especially project-keeper maven plugin and by that fixed report for broken links to opensource.org.
```
Comment (tomuben) [inline doc/changes/changes_2.0.2.md]:
```suggestion
Updated dependencies, especially project-keeper maven plugin and by that fixed check for broken links to opensource.org.
```

**Review by tomuben — COMMENTED** [review]
Check my comments

File: .classpath [hunk 2]
```diff
@@ -15,7 +15,6 @@
 	</classpathentry>
```
Comment (kaklakariada) [inline .classpath]: Remove .classpath and .project and add it to .gitignore.

File: .settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```
Comment (kaklakariada) [inline .settings/org.eclipse.core.resources.prefs]: Delete & ignore this file

File: pom.xml [hunk 4]
```diff
@@ -78,10 +73,17 @@
...
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>project-keeper-maven-plugin</artifactId>
-                <version>1.3.2</version>
+                <version>2.6.2</version>
                 <executions>
                     <execution>
                         <goals>
```
Comment (kaklakariada) [inline pom.xml:88]: Remove `<configuration>` (moved to .project-keeper.yml)

File: .project-keeper.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+sources:
+  - type: maven
+    path: pom.xml
+    modules:
+      - jar_artifact
```
Comment (kaklakariada) [inline .project-keeper.yml]:
```suggestion
```
No need to upload artifact to github release. We deploy to maven central only.

This means you can delete the release_droid_upload_github_release_assets.yml workflow.

File: pom.xml [hunk 6]
```diff
@@ -1,20 +1,15 @@
...
-        framework.
-    </description>
-    <url>https://github.com/exasol/java-util-logging-testing</url>
+        framework.</description>
+    <url>https://github.com/exasol/java-util-logging-testing/</url>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <java.version>11</java.version>
         <junit.version>5.8.1</junit.version>
```
Comment (kaklakariada) [inline pom.xml]:
```suggestion
        <junit.version>5.9.0</junit.version>
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**